contracts/VotingMachines/VotingMachineCallbacks.sol

```diff
@@ -67,4 +67,13 @@ contract GenesisProtocolCallbacks is GenesisProtocolCallbacksInterface {
         return ControllerInterface(avatar.owner()).externalTokenTransfer(_stakingToken,_beneficiary,_amount,address(avatar));
     }
 
+    function balanceOfStakingToken(StandardToken _stakingToken,bytes32 _proposalId) external view returns(uint) {
+        Avatar avatar = proposalsInfo[_proposalId].avatar;
+        if (avatar == Avatar(0)) {
+            return 0;
+        }
+        _stakingToken.balanceOf(address(avatar));
```

Comment:
Shouldn't there be a `return` here?
Also, isn't the `if` redundant?

Reply:
fixed at 9834bf0